app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarFragment.kt

```diff
@@ -116,7 +114,7 @@ class CalendarFragment : FragmentForAccessingTumOnline<EventsResponse>(
                 todayButton.visibility = if (isToday) View.GONE else View.VISIBLE
                 TransitionManager.beginDelayedTransition(swipeRefreshLayout)
```

Comment:
I'm not even sure if this is needed anymore. Ideas on replacing this piece of code?

Reply:
Do you know what this was for? Showing a separate "Today"-Button but hide it when the day is visible?

Reply:
Yes, it was hidden before. I've actually updated it now to always show that button. I don't think its so super useful to hide the button.